### ISSUE TITLE
All contacts are disconected

### ISSUE THREAD

**Issue by funkholio** [EionRobb/purple-teams#56]
![image](https://github.com/EionRobb/purple-teams/assets/25802623/12f8047f-86dc-493a-93dd-04fcbd5eddd5)
All my contacts appear disconected, i can´t write them directly, only respond messages recieved

**Comment by EionRobb** [comment]
Are you able to get a debug log from Help->Debug Window? If you leave it open for approx 15mins there should be a whole bunch of requests asking for user presence status from the server

**Comment by funkholio** [comment]
this?

(07:27:43) http: Performing new request 0992FF40 to apac.ng.msg.teams.microsoft.com.
(07:27:43) teams: Fetching url https://presence.teams.microsoft.com/v1/me/endpoints/
(07:27:43) http: Performing new request 099314E8 to presence.teams.microsoft.com.
(07:27:43) teams: Fetching url https://presence.teams.microsoft.com/v1/me/forceavailability/
(07:27:43) http: Performing new request 099310F8 to presence.teams.microsoft.com.
(07:27:43) teams: Fetching url https://presence.teams.microsoft.com/v1/me/publishnote
(07:27:43) http: Performing new request 099312F0 to presence.teams.microsoft.com.
(07:27:43) teams: Fetching url https://presence.teams.microsoft.com/v1/me/reportmyactivity/
(07:27:43) http: Performing new request 09930FA8 to presence.teams.microsoft.com.
(07:27:43) http: Keep-alive connection expired (when writing), retrying...
(07:27:43) http: Retrying connection 09930C60...
(07:27:43) http: Keep-alive connection expired (when writing), retrying...
(07:27:43) http: Retrying connection 09930918...
(07:27:43) http: Keep-alive connection expired (when reading), retrying...
(07:27:43) http: Retrying connection 09931398...
(07:27:43) http: Keep-alive connection expired (when writing), retrying...
(07:27:43) http: Retrying connection 099310F8...
(07:27:43) http: Keep-alive connection expired (when writing), retrying...
(07:27:43) http: Retrying connection 09931638...
(07:27:43) http: Keep-alive connection expired (when writing), retrying...
(07:27:43) http: Retrying connection 09930C60...
(07:27:43) http: Keep-alive connection expired (when writing), retrying...
(07:27:43) http: Retrying connection 099312F0...
(07:27:43) http: Keep-alive connection expired (when writing), retrying...
(07:27:43) http: Retrying connection 09930918...
(07:27:43) dnsquery: Performing DNS lookup for go.trouter.teams.microsoft.com
(07:27:43) http: Keep-alive connection expired (when writing), retrying...

**Comment by EionRobb** [comment]
Yes, those :)  Did any of those connections work, or did they come back with errors after?

**Comment by funkholio** [comment]
they come back without success

(07:28:04) teams: getting new buddy icon for orgid:f4577609-9102-4ed2-a08c-666f87917980
(07:28:04) http: Performing new request 096F1D80 to teams.microsoft.com.
(07:28:04) teams: getting new buddy icon for orgid:35c4e08e-71ba-48cb-b71e-869f2960ada1
(07:28:04) http: Performing new request 096F1ED0 to teams.microsoft.com.
(07:28:04) http: Request 096F26B0 performed without success.
(07:28:04) http: Request 096F2C98 performed without success.
(07:28:04) teams: getting new buddy icon for orgid:5db88269-6187-471e-bbaa-c2d5b9e8313c
(07:28:04) http: Performing new request 096F1060 to teams.microsoft.com.
(07:28:04) teams: getting new buddy icon for orgid:74089a26-5bb2-4706-8d01-639371ffbabf
(07:28:04) http: Performing new request 096F1A38 to teams.microsoft.com.
(07:28:05) teams: Fetching url https://francecentral-prod.notifications.teams.microsoft.com/users/ME/endpoints/d60142c7-c63e-4e82-b749-ef99bfdc40cc/events/poll?cursor=1720178707&epfs=srt
(07:28:05) http: Performing new request 096F2BF0 to francecentral-prod.notifications.teams.microsoft.com.
(07:28:05) http: Request 096F20C8 performed without success.
(07:28:05) teams: executing callback for https://apac.ng.msg.teams.microsoft.com/v2/users/ME/endpoints/d60142c7-c63e-4e82-b749-ef99bfdc40cc
(07:28:05) teams: Fetching url https://apac.ng.msg.teams.microsoft.com/v1/users/ME/properties
(07:28:05) http: Performing new request 096F2758 to apac.ng.msg.teams.microsoft.com.
(07:28:05) http: Request 096F2AA0 performed without success.
(07:28:05) teams: getting new buddy icon for orgid:58f05e13-9301-4502-b41c-83f5241273ba
(07:28:05) http: Performing new request 096F2020 to teams.microsoft.com.
(07:28:05) http: Request 096F1ED0 performed without success.
(07:28:05) http: Request 096F1D80 performed without success.
(07:28:05) http: Request 096F1060 performed without success.
(07:28:05) http: Request 096F1A38 performed without success.
(07:28:05) http: Request 096F2BF0 performed without success.

**Comment by EionRobb** [comment]
Out of interest, do you connect to the account with the "Away" status in Pidgin?  If so, does it work to set your status to "Available" and then reconnect to the account?

**Comment by funkholio** [comment]
anything?

**Comment by funkholio** [comment]
nope, still not working 
![image](https://github.com/EionRobb/purple-teams/assets/25802623/2a433220-5a7d-41c4-a5e2-b3a885077dee)

**Comment by funkholio** [comment]
no ideas?

**Comment by EionRobb** [comment]
Not yet. The debug log wasn't really helpful, unfortunately. Would you be able to capture more debug while it's going on, leaving the debug window open for at least 5 minutes?

I had it happen to me once, but then when I restarted Pidgin with a debugger attached, it didn't happen again 😥
